frontend/src/lang/es.ts

```diff
@@ -40,6 +40,7 @@ export const es: LanguageConfig = {
         FLUA: "Flu A",
         FLUB: "Flu B",
         RSV: "VRS",
+        HIV: "VIH",
```

Comment:
⚠️ got VIH from the [CDC's usage of it](https://www.cdc.gov/hiv/spanish/index.html)